commands/auth/2fa/generate.js

```diff
@@ -0,0 +1,20 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  let password = yield cli.prompt('Password', {hide: true})
+  let headers = {'Heroku-Password': password}
+  let codes = yield heroku.post('/account/recovery-codes', {headers: headers})
+  cli.log('Recovery codes:')
+  cli.log(codes)
```

Comment:
@RasPhilCo this dumps it in pretty array output like this
```
-> % heroku _auth:2fa:generate 
Password: ***********
Recovery codes:
[ 'aaaa',
  'bbbb']
```
when it should be this
```
-> % heroku _auth:2fa:generate 
Password: ***********
Recovery codes:
aaaa
bbbb
```